server/src/main/java/com/talkka/server/user/dao/UserEntity.java

```diff
@@ -0,0 +1,74 @@
+package com.talkka.server.user.dao;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+@EntityListeners(AuditingEntityListener.class)
+@Table(name = "users")
+public class UserEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "user_id", columnDefinition = "bigint(32)", nullable = false, updatable = false)
+	private Long userId;
+
+	@Column(name = "nickname", columnDefinition = "varchar(50)", nullable = false)
```

Comment:
`VARCHAR` 로 선언하고 `length = 50` 으로 하는 편이 더 나을 것 같습니다. 이후 다른 DB를 사용할 때 조금 더 유용하지 않을까 싶어요.

```suggestion
	@Column(name = "nickname", columnDefinition = "VARCHAR", length = 50, nullable = false)
```

Reply:
그리고 생각해보니 varchar 정도는 따로 표기 안해도 varchar 인 것을 자동으로 알 수 있으니 length 만 표기해도 괜찮을 것 같다는 느낌(??) (정확하지는 않음)

Reply:
해당 내용 확인하여 varchar 컬럼들은 length만 표기하였습니다!